From: Guoqing Jiang <guoqing.jiang@suse.com>
Date: Wed, 22 Dec 2021 16:07:06 +0800
Subject: [PATCH] fix compile issue of intel_pstate

Don't include thermal_interrupt.h for SEL-2.0, because the header file
is introduced in commit 9223d0dccb8f ("thermal: Move therm_throt there
from x86/mce"), and it is not easy to backport the commit given it
touched lots of file.

Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/cpufreq/intel_pstate.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index 453f566a68b9..84ca54e0be76 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -32,7 +32,6 @@
 #include <asm/cpu_device_id.h>
 #include <asm/cpufeature.h>
 #include <asm/intel-family.h>
-#include "../drivers/thermal/intel/thermal_interrupt.h"
 
 #define INTEL_PSTATE_SAMPLING_INTERVAL	(10 * NSEC_PER_MSEC)
 
-- 
2.26.2

